packages/dendron-plugin-views/src/components/SeedBrowser.tsx

```diff
@@ -0,0 +1,97 @@
+import { SeedConfig, SEED_REGISTRY } from "@dendronhq/common-all";
+import { Avatar, Card, Layout, List, PageHeader } from "antd";
+import _ from "lodash";
+import seedStyles from "../styles/scss/seeds.module.scss";
+import { DendronComponent } from "../types";
+import { AddToWorkspaceButton, GoToSiteButton } from "./Seeds";
+
+const SeedBrowser: DendronComponent = (props) => {
+  const { ide, workspace } = props;
+  const { browser } = workspace;
+  const seedDataToRender = _.values(SEED_REGISTRY).map((data) => {
+    if (data) {
+      let payload: SeedConfig & { seedInWorkspace: boolean } = {
+        ...data,
+        seedInWorkspace: false,
+      };
+      if (_.includes(ide.seedsInWorkspace, data?.id)) {
+        payload = {
+          ...payload,
+          seedInWorkspace: true,
+        };
+      }
+      return payload;
+    }
+    return undefined;
+  });
+
+  const { Meta } = Card;
+  const { Content } = Layout;
+
+  return (
+    <>
+      <Layout className={seedStyles.layout}>
+        <Content style={{ padding: "0 50px" }}>
+          <div className={seedStyles.contentDiv}>
+            <PageHeader
+              className={seedStyles.siteHeader}
+              title="Dendron Seed Registry"
+              subTitle="Add Knowledge Bases to your Workspace"
+            />
+            <div className={seedStyles.listDiv}>
+              <List
+                grid={{
+                  gutter: 24,
+                  xs: 1,
+                  sm: 2,
+                  md: 3,
+                  lg: 3,
+                  xl: 4,
+                  xxl: 4,
+                }}
+                dataSource={seedDataToRender}
+                renderItem={(item) => (
+                  <List.Item>
+                    <Card
+                      className={seedStyles.card}
+                      hoverable
+                      actions={[
+                        <GoToSiteButton
```

Comment:
both action buttons currently have a `.` next to the icon > https://www.loom.com/i/bc2fef4657ba4625935c13054a73d737